src/Filters/molecules/FilterDropdown.tsx

```diff
@@ -38,12 +38,22 @@ const OptionList = styled.div`
   max-height: 162px;
   min-height: 40px;
   position: relative;
-  overflow-y: scroll;
-  ::-webkit-scrollbar {  /* Hide scrollbar for Chrome, Safari and Opera */
-    display: none;
+  overflow-y: auto;
+  margin-right: 5px;
+
+  /* Customize the scrollbar */
+  ::-webkit-scrollbar {
+    width: 6px;
+    height: 34%;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    border-radius: 10px;
+    background: #44444487;
+    opacity: 0.5;
+    transform: translate(0px, 0px);
+    transition: background-color .3s;
```

Comment:
I think you want to apply some hover effect on **background** as you have used `transition`, but there is no such feature applied, so I think if you don't need it you can remove it.